app/Resources/views/newsletter/invitation_sent.html.twig

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html.twig' %}
+
+{% block banner '' %}
+{% block footer '' %}
+
+{% block page_title 'Vous êtes désormais inscrit à notre newsletter' %}
+{% block canonical_url url('newsletter_subscription') %}
+
+{% block content %}
+
+    <main class="newsletter-result">
+        <h2>Votre(vos) invitation(s) a(ont) bien été envoyée(s) !</h2>
```

Comment:
Let's be pragamatic: the current solution (without adaptation) will be enough for now